README.md

```diff
@@ -66,13 +66,14 @@ Customer A|Project1;True;True;;;
 Customer A|Project2;false;true;;2019-05-16;
 ```
 
-Optionally Custom Fields can be included, which then requires the following format:
+Optionally Custom Fields can be included, which then requires the following format
+Optionally Subtasks can be included. They need to be seperated by ','
```

Comment:
I probably would split it up in an extra section and add more info about how the subtasks are handled (e.g. billable, bookable)